community/kernel/src/main/java/org/neo4j/kernel/TransactionBuilder.java

```diff
@@ -19,33 +19,12 @@
  */
 package org.neo4j.kernel;
 
+import org.neo4j.graphdb.GraphDatabaseService;
 import org.neo4j.graphdb.Transaction;
 
 /**
- * A builder for controlling certain behaviors of a transaction.
- * Methods that return {@link TransactionBuilder} returns a new instance
- * with the updated configuration that the method call modified.
- * When the behavior is as desired a call to {@link #begin()} will
- * begin the transaction and return a {@link Transaction}.
- *
- * <pre>
- * // To begin a transaction with default behavior
- * Transaction tx = graphDb.tx().begin();
- *
- * // To begin a transaction with relaxed force
- * Transaction tx = graphDb.tx().unforced().begin();
```

Comment:
It has been deprecated a while now. You changed the deprecation description, but I think you can simply delete this class alltogether. No one will ever miss it.